test/dialect/postgresql/test_reflection.py

```diff
@@ -1582,6 +1582,32 @@ def test_reflect_check_warning(self):
             ):
                 testing.db.dialect.get_check_constraints(conn, "foo")
 
+    def test_reflect_extra_newlines(self):
+        rows = [
+            ("some name", "CHECK (\n(a \nIS\n NOT\n\n NULL\n)\n)"),
+            ("some other name", "CHECK ((b\nIS\nNOT\nNULL))"),
+            ("some CRLF name", "CHECK ((c\r\n\r\nIS\r\nNOT\r\nNULL))"),
+        ]
+        conn = mock.Mock(
+            execute=lambda *arg, **kw: mock.MagicMock(
+                fetchall=lambda: rows, __iter__=lambda self: iter(rows)
+            )
+        )
+        with mock.patch.object(
+            testing.db.dialect, "get_table_oid", lambda *arg, **kw: 1
+        ):
+            check_constraints = testing.db.dialect.get_check_constraints(
+                conn, "foo"
+            )
+            eq_(
+                check_constraints,
+                [
+                    {"name": "some name", "sqltext": "a  IS  NOT   NULL ",},
+                    {"name": "some other name", "sqltext": "b IS NOT NULL",},
+                    {"name": "some CRLF name", "sqltext": "c  IS NOT NULL",},
+                ],
```

Comment:
**Gord Thompson** (gordthompson) wrote:

"E231 Missing whitespace after ','" - I ran into this before:

https://gerrit.sqlalchemy.org/#/c/sqlalchemy/sqlalchemy/+/1706/6/test/dialect/mssql/test_types.py@802

@zzzeek - Can we get Black and flake8 to agree on this somehow?

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/1749

Reply:
you can manually remove the comma, black will not re-add them